aait/backend/group-2B/WebApi/Services/UserService.cs

```diff
@@ -12,9 +12,9 @@ public UserService(IHttpContextAccessor httpContextAccessor)
         _httpContextAccessor = httpContextAccessor;
     }
 
-    public string GetUserId()
+    public int GetUserId()
     {
         var userId = _httpContextAccessor.HttpContext?.User.FindFirst(ClaimTypes.NameIdentifier)?.Value;
-        return userId!;
+        return int.Parse(userId);
```

Comment:
It'll never happen, but the userId might be null if the user is not authenticated and/or authorized, so better add a null safety in the return statement.